### DISCUSSION TITLE
What is the difference between `space f` and `space F`?

### DISCUSSION THREAD

**Discussion by sporto** [helix-editor/helix#5759] [Q&A] [answered]
For me these two combinations always open the same file list, regardless of the file I have opened.

Unsure what `working directory` means here.

For example if I currently have a file like `a/b/file.md`
Should `space F` show me files only in directory `a/b`?

If that not the case, is there a way to do that?

**Comment by gboncoffee** [comment, marked as answer]
`space f` will show files from the git workspace if available (e.g., if the current dir is foo/bar but foo is a git repo, it will show everything from foo). `space F` will show files from the current directory ($PWD) only, even if it's children of a git repo.

**Comment by sporto** [comment]
Oh, I see thanks.
I open helix at the root of the git repository, so I see no difference.

Is there maybe a way to list files on same directory as the current opened file?
Thanks

**Reply by gboncoffee** [reply]
You're welcome. As far as I know, there isn't.

**Reply by gabydd** [reply]
actually this just got merged today: https://github.com/helix-editor/helix/pull/4666 its not bound by default

**Reply by gboncoffee** [reply]
Excelent! Anyway, I'm following releases so I won't get it for a while.
